Travis CI build is failing

The site is currently not building through Travis-CI and the build badge on the README is set to `failing`. I am also unable to get the build functional on my OSX10.14.

The error I'm getting on my machine is:

```
Traceback (most recent call last):
  File "generator.py", line 624, in <module>
    build(build_images=args.images, to_update=get_flat_names_to_update(all_=args.all, random=args.random, files=args.files))
  File "generator.py", line 541, in build
    build_reference_index(reference_dir, env)
  File "generator.py", line 439, in build_reference_index
    str(set(categories.keys()) - set(category_order))
AssertionError: category order and category keys are different. set([('Math', '')]) *** set([('Math', 'PVector')])
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,3 +1,5 @@
+dist: trusty
+
 env: DISPLAY=:99.0 # from p5py
 
 language: java # say it's Java so we can use the jdk

--- a/Reference/api_en/fullScreen.xml
+++ b/Reference/api_en/fullScreen.xml
@@ -9,7 +9,7 @@
 <type></type>
 
 <example>
-<notest/>
+<notest /><noimage />
 <code><![CDATA[
 # Run the code at the full dimensions of the screen currently
 # selected inside the Preferences window
@@ -30,6 +30,7 @@ def draw():
 </example>
 
 <example>
+<notest /><noimage />
 <code><![CDATA[
 # If more than one screen is attached to the computer, run the 
 # code at the full dimensions on the screen defined by the 
@@ -51,6 +52,7 @@ def draw():
 </example>
 
 <example>
+<notest /><noimage />
 <code><![CDATA[
 # Run full screen using the P2D renderer on screen 2
 
@@ -70,6 +72,7 @@ def draw():
 </example>
 
 <example>
+<notest /><noimage />
 <code><![CDATA[
 # If more than one screen is attached to the computer, run the 
 # code at the full dimensions across all of the attached screens

--- a/Reference/api_en/fullScreen.xml
+++ b/Reference/api_en/fullScreen.xml
@@ -9,7 +9,6 @@
 <type></type>
 
 <example>
-<image></image>
 <notest/>
 <code><![CDATA[
 # Run the code at the full dimensions of the screen currently
@@ -31,7 +30,6 @@ def draw():
 </example>
 
 <example>
-<image></image>
 <code><![CDATA[
 # If more than one screen is attached to the computer, run the 
 # code at the full dimensions on the screen defined by the 
@@ -53,7 +51,6 @@ def draw():
 </example>
 
 <example>
-<image></image>
 <code><![CDATA[
 # Run full screen using the P2D renderer on screen 2
 
@@ -73,7 +70,6 @@ def draw():
 </example>
 
 <example>
-<image></image>
 <code><![CDATA[
 # If more than one screen is attached to the computer, run the 
 # code at the full dimensions across all of the attached screens

--- a/generator.py
+++ b/generator.py
@@ -425,7 +425,7 @@ def build_reference_index(reference_dir, env):
         ('Typography', 'Loading & Displaying'),
         ('Typography', 'Attributes'),
         ('Typography', 'Metrics'),
-        ('Math', ''),
+        ('Math', 'PVector'),
         ('Math', 'Operators'),
         ('Math', 'Bitwise Operators'),
         ('Math', 'Calculation'),

